shared/searchv3/services-filter/index.js

```diff
@@ -118,4 +127,20 @@ class Filter extends Component<void, Props, {hoveredService: ?Constants.Service}
   }
 }
 
+const serviceTooltipStyle = {
+  ...transition('opacity'),
+  backgroundColor: globalColors.black_40,
+  borderBottom: '2px solid white',
+  borderRadius: 65,
+  color: globalColors.white,
```

Comment:
this isn't centered correctly

![image](https://user-images.githubusercontent.com/516435/27647698-2e62e34c-5bfa-11e7-873d-3d080e180487.png)